entity-registry/src/main/java/com/linkedin/metadata/models/registry/LineageRegistry.java

```diff
@@ -86,6 +89,14 @@ public LineageSpec getLineageSpec(String entityName) {
     return _lineageSpecMap.get(entityName.toLowerCase());
   }
 
+  public Set<String> getEntitiesWithLineage() {
+    Map<String, EntitySpec> specs = _entityRegistry.getEntitySpecs();
+    return _lineageSpecMap.keySet().stream().filter(key ->
```

Comment:
Minor: You maybe could use _lineageSpecMap.entrySet() to avoid the .get(key) calls